src/AspNetStandard.Diagnostics.HealthChecksWcf/AspNetStandard.Diagnostics.HealthChecks.Wcf.csproj

```diff
@@ -0,0 +1,74 @@
+    <OutputType>Library</OutputType>
+    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <RootNamespace>AspNetStandard.Diagnostics.HealthChecksWcf</RootNamespace>
+    <AssemblyName>AspNetStandard.Diagnostics.HealthChecksWcf</AssemblyName>
+    <TargetFrameworkVersion>v4.6.1</TargetFrameworkVersion>
+    <FileAlignment>512</FileAlignment>
+    <Deterministic>true</Deterministic>
+    <TargetFrameworkProfile />
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
+    <DebugSymbols>true</DebugSymbols>
+    <DebugType>full</DebugType>
+    <Optimize>false</Optimize>
+    <OutputPath>bin\Debug\</OutputPath>
+    <DefineConstants>DEBUG;TRACE</DefineConstants>
+    <ErrorReport>prompt</ErrorReport>
+    <WarningLevel>4</WarningLevel>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
+    <DebugType>pdbonly</DebugType>
+    <Optimize>true</Optimize>
+    <OutputPath>bin\Release\</OutputPath>
+    <DefineConstants>TRACE</DefineConstants>
+    <ErrorReport>prompt</ErrorReport>
+    <WarningLevel>4</WarningLevel>
+  </PropertyGroup>
+  <ItemGroup>
+    <Reference Include="System" />
+    <Reference Include="System.Configuration" />
+    <Reference Include="System.Core" />
+    <Reference Include="System.Runtime.Serialization" />
+    <Reference Include="System.ServiceModel" />
+    <Reference Include="System.ServiceModel.Activation" />
+    <Reference Include="System.ServiceModel.Web" />
+    <Reference Include="System.Web" />
+    <Reference Include="System.Xml.Linq" />
+    <Reference Include="System.Data.DataSetExtensions" />
+    <Reference Include="Microsoft.CSharp" />
+    <Reference Include="System.Data" />
+    <Reference Include="System.Net.Http" />
+    <Reference Include="System.Xml" />
+  </ItemGroup>
+  <ItemGroup>
+    <Compile Include="Entities\WcfHealthCheckResponse.cs" />
+    <Compile Include="WcfHealthCheckService.cs" />
+    <Compile Include="WcfHealthCheckBuilder.cs" />
+    <Compile Include="WcfHealthCheck.cs" />
+    <Compile Include="WcfHealthCheckConfiguration.cs" />
+    <Compile Include="Properties\AssemblyInfo.cs" />
+    <Compile Include="WcfHeathCheckRouteExtension.cs" />
+    <Compile Include="Entities\WcfHealthCheckResult.cs" />
+  </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\AspNetStandard.Diagnostics.HealthChecks.SqlServer\AspNetStandard.Diagnostics.HealthChecks.SqlServer.csproj">
+      <Project>{A008AA4D-6AEC-4901-B0C5-EAD907AF146F}</Project>
+      <Name>AspNetStandard.Diagnostics.HealthChecks.SqlServer</Name>
+    </ProjectReference>
+    <ProjectReference Include="..\AspNetStandard.Diagnostics.HealthChecks\AspNetStandard.Diagnostics.HealthChecks.csproj">
+      <Project>{8474BE3A-E5B4-4E6B-A9BC-BDBF28466B52}</Project>
+      <Name>AspNetStandard.Diagnostics.HealthChecks</Name>
+    </ProjectReference>
+  </ItemGroup>
+  <ItemGroup>
+    <None Include="README.md" />
+  </ItemGroup>
```

Comment:
Deve incluir esse item group pra passar na pipeline de CI CD

	<ItemGroup Condition="$(TargetFramework.StartsWith('net4')) AND '$(MSBuildRuntimeType)' == 'Core' AND '$(OS)' != 'Windows_NT'">
		<PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.0" PrivateAssets="All" />
	</ItemGroup>